.gitignore

```diff
@@ -85,6 +85,7 @@ _ReSharper*/
 *.resharper
 [Tt]est[Rr]esult*
 *.sass-cache
+WiniumDriver/
```

Comment:
A powershell script was created and located in the Playground.Net46 directory that will run the Winium tests outside of AppVeyor... That script creates a folder called WiniumDriver...